Assets/Scripts/2- Managers/NotesInteractor.cs

```diff
@@ -15,9 +15,13 @@ void Start()
             _camera = GetComponent<Camera>();
         }
 
-        void Update()
+        void LateUpdate()
         {
-            if (Physics.Raycast(_camera.ViewportToWorldPoint(new Vector3(0.5f, 0.5f)), transform.forward, out RaycastHit hit, rayLength))
+
+            Vector3 origin = _camera.ViewportToWorldPoint(new Vector3(0.5f, 0.5f, _camera.nearClipPlane));
+            Ray ray = new Ray(origin, transform.forward);
```

Comment:
doing a new like this in an update function will cause a lot of allocations. Ideally save a ray object local to the class and update it in here.